src/ert/_c_wrappers/enkf/res_config.py

```diff
@@ -28,6 +29,22 @@ def format_warning_message(message, category, *args, **kwargs):
 warnings.formatwarning = format_warning_message
 
 
+def site_config_location():
```

Comment:
You could be excused for thinking so, because of the names. But, site_config.py is not a concept that has much to do with the site_config file, except in so far as they are both somewhat related to configuring some stuff. Also, site_config.py is going away ASAP, because it really serves no purpose other than obfuscation.

Reply:
On another note, I do also think that the way we figure out location of certain files has little to do with reading the contents of the files.